modules/tinymce/src/plugins/accordion/test/ts/browser/AccordionPluginTest.ts

```diff
@@ -121,12 +130,154 @@ describe('browser.tinymce.plugins.accordion.AccordionPluginTest', () => {
+    const editor = hook.editor();
+    editor.setContent(`${createAccordion()}<p>tiny</p>`);
+    TinySelections.setCursor(editor, [ 0, 0, 0 ], 'tiny'.length);
+    editor.execCommand('RemoveAccordion');
+    TinyAssertions.assertContent(editor, '<p>tiny</p>');
+    TinyAssertions.assertCursor(editor, [ 0, 0 ], 0);
+  });
+
+  it('TINY-9731: Toggle an accordion element under the cursor', () => {
+    const editor = hook.editor();
+    editor.setContent(`${createAccordion({ open: true })}<p>tiny</p>`);
+    TinySelections.setCursor(editor, [ 0, 0, 0 ], 0);
+    editor.execCommand('ToggleAccordion');
+    TinyAssertions.assertContentPresence(editor, { 'details:not([open="open"])': 1 });
+    TinyAssertions.assertCursor(editor, [ 0, 0, 0 ], 0);
+    editor.execCommand('ToggleAccordion');
+    TinyAssertions.assertContentPresence(editor, { 'details[open="open"]': 1 });
+    TinyAssertions.assertCursor(editor, [ 0, 0, 0 ], 0);
+  });
+
+  it('TINY-9731: Toggle an accordion element under the cursor with an argument', () => {
+    const editor = hook.editor();
+    editor.setContent(`${createAccordion({ open: true })}<p>tiny</p>`);
+    TinySelections.setCursor(editor, [ 0, 0, 0 ], 0);
+    editor.execCommand('ToggleAccordion', false, false);
+    TinyAssertions.assertContentPresence(editor, { 'details:not([open="open"])': 1 });
+    TinyAssertions.assertCursor(editor, [ 0, 0, 0 ], 0);
+    editor.execCommand('ToggleAccordion', false, true);
+    TinyAssertions.assertContentPresence(editor, { 'details[open="open"]': 1 });
+    TinyAssertions.assertCursor(editor, [ 0, 0, 0 ], 0);
+    editor.execCommand('ToggleAccordion', false, true);
+    TinyAssertions.assertContentPresence(editor, { 'details[open="open"]': 1 });
+    TinyAssertions.assertCursor(editor, [ 0, 0, 0 ], 0);
+  });
+
+  it('TINY-9731: Toggle all accordion elements', () => {
+    const editor = hook.editor();
+    editor.setContent([ createAccordion({ open: true }), createAccordion({ open: true }) ].join(''));
+    TinySelections.setCursor(editor, [ 0, 0, 0 ], 0);
+    editor.execCommand('ToggleAllAccordions');
+    TinyAssertions.assertContentPresence(editor, { 'details:not([open="open"])': 2 });
+    TinyAssertions.assertCursor(editor, [ 0, 0, 0 ], 0);
+    editor.execCommand('ToggleAllAccordions');
+    TinyAssertions.assertContentPresence(editor, { 'details[open="open"]': 2 });
+    TinyAssertions.assertCursor(editor, [ 0, 0, 0 ], 0);
+  });
+
+  it('TINY-9731: Toggle all accordion elements with an argument', () => {
+    const editor = hook.editor();
+    editor.setContent([ createAccordion({ open: true }), createAccordion({ open: true }) ].join(''));
+    TinySelections.setCursor(editor, [ 0, 0, 0 ], 0);
+    editor.execCommand('ToggleAllAccordions', false, false);
+    TinyAssertions.assertContentPresence(editor, { 'details:not([open="open"])': 2 });
+    TinyAssertions.assertCursor(editor, [ 0, 0, 0 ], 0);
+    editor.execCommand('ToggleAllAccordions', true, true);
```

Comment:
Apparently no UI involved for this command
```suggestion
    editor.execCommand('ToggleAllAccordions', false, true);
```

Reply:
Fixed.